module1/presentation_catch.md

```diff
@@ -51,11 +51,14 @@ ___
 
 ## Organizacja testów
 
-* <!-- .element: class="fragment fade-in" --> <code>TEST_CASE( test name [, tags ] )</code>
+* <!-- .element: class="fragment fade-in" --> <code>TEST_CASE( test name , [ tags ] )</code>
```

Comment:
To akurat powinno zostać tak jak było :)
To co w nawiasach jest opcjonalne, więc jeśli nie ma tagów to i przecinka nie będzie. W wielu dokumentacjach stosuje się tego typu zapis.
Dla SCENARIO będzie podobnie.